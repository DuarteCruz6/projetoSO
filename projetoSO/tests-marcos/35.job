WAIT 3000
SHOW
WRITE [(l7,61)(w7,43)]
WRITE [(k2,75)(d4,100)(e3,81)(l9,99)]
WRITE [(p0,89)(e4,59)(u0,35)]
WRITE [(s1,41)(a5,65)]
BACKUP
WRITE [(p4,79)(o9,11)(k3,47)(y3,52)]
WRITE [(k8,16)]
WRITE [(i2,54)(n8,90)(i0,72)(e7,36)]
WRITE [(i5,40)(n2,46)(n4,4)]
WRITE [(t5,13)(q2,72)]
SHOW
READ [a5]
WAIT 2000
WRITE [(b8,16)(c6,92)(u6,88)(z2,43)(b1,80)]
READ [e4]
WRITE [(h5,54)(t0,63)]
DELETE [i5,e3]
WRITE [(s0,38)]
WRITE [(n1,47)(z0,84)(x5,7)(u4,64)(i4,25)]
DELETE [k8,l7,u6,w7]
WRITE [(f5,30)]
DELETE [b1,e4,n1,s0]
WRITE [(r7,66)(g6,43)(m4,67)(u8,66)(d5,51)]
READ [u0]
WRITE [(m3,49)(g2,59)]
WRITE [(i7,25)(x8,39)(z9,36)]
WRITE [(j8,4)(u5,58)(t1,12)]
WRITE [(d6,4)(a9,3)(y5,48)]
WRITE [(o2,16)(l8,43)(r3,9)(f4,47)(c9,31)]
READ [r3]
READ [x5]
DELETE [i2,j8,n8]
WRITE [(d8,58)(o5,88)]
READ [o2]
READ [o2]
SHOW
WAIT 3000
WRITE [(h8,100)(c3,13)(w6,41)(f9,7)]
READ [i7]
