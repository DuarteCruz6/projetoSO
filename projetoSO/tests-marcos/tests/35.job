WRITE [(m2,64)(u7,65)(y3,30)(q0,25)]
WRITE [(k1,11)]
WRITE [(q2,2)(j9,79)(n9,45)(i9,7)]
READ [k1]
WRITE [(a7,42)(s2,56)(e5,80)(o1,19)(y7,85)]
WRITE [(k7,11)]
WRITE [(l7,46)(w2,17)]
SHOW
DELETE [k7,l7,m2]
WRITE [(i1,18)(d9,21)(r4,52)]
SHOW
WRITE [(p0,30)(w7,4)(m8,58)]
WAIT 3000
WRITE [(a4,62)]
DELETE [q2]
READ [w2]
WRITE [(n0,43)]
WRITE [(s0,31)(k7,79)(j1,64)]
READ [u7]
WRITE [(h4,76)]
WRITE [(m9,57)(m3,54)(p2,34)(z6,38)(h5,47)]
WRITE [(l4,51)(v9,53)]
READ [s0]
DELETE [p0,m3,w7]
WRITE [(b9,34)]
WRITE [(y0,35)(q1,15)(c7,100)(p6,66)(u0,9)]
DELETE [v9,b9,o1,c7]
WRITE [(a8,93)]
WRITE [(v4,17)(u6,82)(o9,67)]
WRITE [(i3,7)(r8,87)(a2,43)(g1,53)]
DELETE [a2,z6,y3,a8,a7]
WRITE [(c9,77)(n1,50)(o6,36)]
WRITE [(j3,29)(o4,59)(s4,46)(a9,4)(l3,73)]
WRITE [(f5,87)(i6,46)]
WRITE [(j4,29)]
