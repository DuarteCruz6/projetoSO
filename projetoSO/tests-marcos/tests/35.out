[(k1,11)]
(a7, 42)
(e5, 80)
(i9, 7)
(j9, 79)
(k7, 11)
(k1, 11)
(l7, 46)
(m2, 64)
(n9, 45)
(o1, 19)
(q2, 2)
(q0, 25)
(s2, 56)
(u7, 65)
(w2, 17)
(y7, 85)
(y3, 30)
(a7, 42)
(d9, 21)
(e5, 80)
(i1, 18)
(i9, 7)
(j9, 79)
(k1, 11)
(n9, 45)
(o1, 19)
(q2, 2)
(q0, 25)
(r4, 52)
(s2, 56)
(u7, 65)
(w2, 17)
(y7, 85)
(y3, 30)
[(w2,17)]
[(u7,65)]
[(s0,31)]
